tss-esapi/tests/integration_tests/common/tpms_types_equality_checks.rs

```diff
@@ -223,3 +224,15 @@ pub fn ensure_tpms_tagged_property_equality(
         "'value' value in TPMS_TAGGED_PROPERTY, mismatch between actual and expected",
     );
 }
+
+#[allow(dead_code)]
```

Comment:
Why is the `#[allow(dead_code)]` present here? The code looks straightforward... 🤔

Reply:
I had some issues with the compiler not understand that the functions were in use in the tests.

Reply:
Fixed